### DISCUSSION TITLE
Debby: a tool to work with versioned data using visidata and dolt

### DISCUSSION THREAD

**Discussion by LaPingvino** [saulpw/visidata#2039] [Show and tell]
I created a tool at https://codeberg.org/lapingvino/debby that enables you to version your data in dolt, a version controlled mysql compatible database that follows Git semantics, using visidata as your table editor. Please give it a try and let me know what you think!

**Comment by daviewales** [comment]
Now here: https://git.kiefte.eu/debby.git/about/

**Reply by LaPingvino** [reply]
Better use codeberg.org/lapingvino/debby instead. Edited the starting post.
